manifests/d/Dyad/Dyad/0.21.0/Dyad.Dyad.locale.en-US.yaml

```
@@ -0,0 +1,38 @@
+# Created with komac v2.13.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.10.0.schema.json
+
+PackageIdentifier: Dyad.Dyad
+PackageVersion: 0.21.0
+PackageLocale: en-US
+Publisher: Will Chen
+PublisherUrl: https://www.dyad.sh/
+PublisherSupportUrl: https://github.com/dyad-sh/dyad/issues
+Author: Dyad
+PackageName: Free, local, open-source AI app builder

```

```suggestion
PackageName: dyad
```

Should match the name written to Control Panel / Apps&Features